cpp/src/Ice/Network.cpp

```diff
@@ -291,10 +284,9 @@ namespace
                     {
                         Address addr;
                         memcpy(&addr.saStorage, &ifr[j].ifr_addr, sizeof(sockaddr_in));
-                        if (addr.saIn.sin_addr.s_addr != 0 &&
```

Comment:
All this code is apparently for Solaris and AIX. I think we should remove it.

Reply:
I agree we should just remove all this.
Definitely seems like it's just leftover from Solaris support...